packages/runtime-dom/src/modules/events.ts

```diff
@@ -132,19 +132,11 @@ function createInvoker(
     // AFTER it was attached.
     if (e.timeStamp >= invoker.lastUpdated - 1) {
       const args = [e]
-      const value = invoker.value
-      if (isArray(value)) {
-        for (let i = 0; i < value.length; i++) {
-          callWithAsyncErrorHandling(
-            value[i],
-            instance,
-            ErrorCodes.NATIVE_EVENT_HANDLER,
-            args
-          )
-        }
-      } else {
+      let {value} = invoker
+      value = isArray(value) ? value : [value]
+      for (const item of value) {
```

Comment:
But `for...of` is extremely slow. See this benchmark: https://jsperf.com/repeat-vs-normalize